src/main/java/songbox/house/domain/entity/user/UserProperty.java

```diff
@@ -0,0 +1,29 @@
+package songbox.house.domain.entity.user;
+
+import lombok.Getter;
+import lombok.Setter;
+import songbox.house.domain.entity.MusicCollection;
+
+import javax.persistence.*;
+
+import static javax.persistence.CascadeType.ALL;
+
+@Table
+@Entity
+@Getter
+@Setter
+public class UserProperty {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column
+    private Long userPropId;
+
+    @Column
+    private Boolean telegramBotUseGoogleDrive;
+
+    @Column(length = 2048)
+    private String vkCookie;
+
+    @OneToOne(cascade = ALL, fetch = FetchType.LAZY)
+    private MusicCollection defaultCollection;
```

Comment:
cascade = ALL means that if you remove collection -> User property will be removed as well(need to check, maybe it works reversed - if you remove property -> collections will be removed as well)

Anyway, I think we don't need such behavior